### ISSUE TITLE
vimeo on demand - downloads only a few MiBs out of the full video

### ISSUE THREAD

**Issue by kaliaragorn** [rg3/youtube-dl#9830]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2016.06.19*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2016.06.19**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [x] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [x] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add `-v` flag to **your command line** you run youtube-dl with, copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):
```
$ youtube-dl.exe  --verbose https://vimeo.com/ondemand/76870/ -u XXX  -p YYY
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'https://vimeo.com/ondemand/76870/', '-u', 'PRIVATE', '-p', 'PRIVATE']
[debug] Encodings: locale cp1252, fs mbcs, out cp1252, pref cp1252
[debug] youtube-dl version 2016.06.19
[debug] Python version 3.4.4 - Windows-10-10.0.10586
[debug] exe versions: none
[debug] Proxy map: {}
[vimeo:ondemand] 76870: Downloading webpage
[vimeo] Logging in
[vimeo] 169738107: Downloading webpage
[vimeo] 169738107: Extracting information
[vimeo] 169738107: Downloading JSON metadata
[vimeo] 169738107: Downloading m3u8 information
[debug] Invoking downloader on 'https://fpdl.vimeocdn.com/vimeo-prod-skyfire-std-us/01/3947/6/169738107/544648336.mp4?token=5765be1f_0x0ef9b02b573270139d343e87eb95391fd35846c6'
[download] Destination: Sebastian Kr▒ger Studios-169738107.mp4
[download] 100% of 5.16MiB in 00:06

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://vimeo.com/ondemand/76870/

---

### Description of your *issue*, suggested solution and other information

instead of the full video the system keeps downloading a ~5Mib preamble.

credentials: <snip>

**Comment by dstftw** [comment]
You have just posted your credentials on a public forum so you may want to change them.
Rented video has been fixed in youtube-dl 2016.06.19.1.

**Comment by kaliaragorn** [comment]
excellent! it seems to work perfectly now. thank you! :) :+1:
